$NetBSD: patch-Makefile,v 1.1 2014/09/08 15:12:24 thomasklausner Exp $

Use LDFLAGS so rpath can be passed in from pkgsrc.

--- Makefile.orig	2005-04-18 20:26:33.000000000 +0000
+++ Makefile
@@ -37,10 +37,10 @@ libptformats.a:     $(LIBFORMATSOBJ)
 
 
 pbm_findskew/pbm_findskew:  pbm_findskew/pbm_findskew.o $(PTLIB)
-	$(C++) pbm_findskew/pbm_findskew.o $(PTLIB) -l$(LIBPBM) -o $@
+	$(C++) pbm_findskew/pbm_findskew.o $(LDFLAGS) $(PTLIB) -l$(LIBPBM) -o $@
 
 tiff_findskew/tiff_findskew:  tiff_findskew/tiff_findskew.o $(PTLIB)
-	$(C++) tiff_findskew/tiff_findskew.o $(PTLIB) -ltiff -o $@
+	$(C++) tiff_findskew/tiff_findskew.o $(LDFLAGS) $(PTLIB) -ltiff -o $@
 
 
 clean:
